subcommands/keys/common.go

```diff
@@ -138,7 +239,7 @@ func GetOfflineCreds(credsFile string) (OfflineCreds, error) {
 	return files, nil
 }
 
-func FindPrivKey(pubkey string, creds OfflineCreds) (*rsa.PrivateKey, error) {
+func FindSigner(keyid, pubkey string, creds OfflineCreds) (*TufSigner, error) {
```

Comment:
I can see you've had HSM's on your mind :)

Reply:
Actually, just trying to find a common demoninator between Golang's crypto/xyz libraries.
In general they are almost incompatible; those `crypto.Signer` interface and `x509.XxxPKCS8yyy` routines are the only things that made my life a little bit easier.